pom.xml.versionsBackup

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
Please don't commit the versions backup file.  It needs to be removed from the pull request.  The tutorial has been updated to use `git add pom.xml` instead of `git add .` so that it will avoid this mistake in the future.